### DISCUSSION TITLE
Signing with Session Key not working

### DISCUSSION THREAD

**Discussion by Karmabir-Brahma** [bcnmy/biconomy-client-sdk#487] [Q&A]
![Screenshot (272)](https://github.com/bcnmy/biconomy-client-sdk/assets/98337002/5b4416ce-8608-46f0-9862-1f46c9ef8d8e)

Followed Session Key Tutorial from(SDK V4). But facing AA23(OOG) error even though I have some amoy tokens in my Smart Account. I m  trying to execute some function but not for any token transfer(like ERC20).

**Comment by himanshugarg06** [comment]
are you using SPONSORED mode of paymaster? also did you check for the error here on the troubleshooting page  - https://docs.biconomy.io/troubleshooting/commonerrors
